web-tech-exercises/2/clock.js

```diff
@@ -0,0 +1,22 @@
+function updateClock() {
+    const clock = document.getElementById('clock');
+    const now = new Date();
+    const hours = String(now.getHours()).padStart(2, '0');
+    const minutes = String(now.getMinutes()).padStart(2, '0');
+    const seconds = String(now.getSeconds()).padStart(2, '0');
```

Comment:
try using to toLocaleTimeString https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

Reply:
Done .✅